doc/source/admin/interactive_environments.rst

```diff
@@ -3,21 +3,22 @@ Galaxy Interactive Environments (GIEs)
 
 GIEs were a new feature back in Galaxy 15.05, leading with the release of the
 IPython IE. They were presented at GCC2015, and the RStudio IE was released as
-part of 15.07.
+part of 15.07. The IPython IE was superceded by a Project Jupyter IE in
+Galaxy XXXXX.
 
-A GIE is a docker container, launched by Galaxy, proxied by Galaxy, with some
+A GIE is a Docker container, launched by Galaxy, proxied by Galaxy, with some
 extra sugar inside the container to allow users to interact easily with their
 Galaxy histories.
 
 How GIEs Work
 -------------
 
 A GIE is primarily composed of a Docker container, and the Galaxy visualization
-component. GIE's are essentially visualization plugins which abuse the fact
-that they're rendered with mako templates. The mako templating language used in
-Viz plugins allows running python code, which in turn allows for us to launch
-docker containers. Once this container is launched, we notify a proxy built
-into Galaxy which helps coordinate a 1:1 mapping of users and their docker containers.
+component. Galaxy vizualisation plugins are rendered using Mako templates and
+Mako templates in turn can run Python code. This Python code in the Mako templates
```

Comment:
"code. This" -> "code. GIEs build upon visualization plugins, adding features to allow for container management and proxying. This"